lib/fog/aws/models/storage/file.rb

```diff
@@ -1,4 +1,5 @@
 require 'fog/aws/models/storage/versions'
+require 'concurrent-ruby'
```

Comment:
Won't this need to be added to the Gemfile and other dependencies? I'm a little reticent to add dependencies, but this one looks well vetted (and doesn't appear to have sub-dependencies to contend with).

Reply:
@geemus For some reason, I thought `concurrent-ruby` was already included, but now I see that is my local `Gemfile.lock`. 😄 

I got rid of this dependency with a thread-safe queue that is similar to the AWS SDK.